apps/meteor/server/models/raw/Rooms.js

```diff
@@ -62,7 +62,7 @@ export class RoomsRaw extends BaseRaw {
 		return statistic;
 	}
 
-	findByNameContainingAndTypes(name, types, discussion = false, teams = false, showOnlyTeams = false, options = {}) {
+	findByNameOrFnameContainingAndTypes(name, types, discussion = false, teams = false, showOnlyTeams = false, options = {}) {
```

Comment:
When you change a function name you need to change all the places that uses it

Reply:
I accidentaly forgot to add this file in the commit. Thanks!